src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs

```diff
@@ -149,7 +151,9 @@
                                                    bucketName "/" submission-id "/"
                                                    workflow-name "/" (workflow "id") "/")})))
     (let [status (workflow "status")]
-      (create-field "Status" (moncommon/icon-for-wf-status status) status))
+      (create-field "Status" (moncommon/icon-for-wf-status status)))
+    (let [call-cache-status (get (first (first (vals (workflow "calls")))) "Effective call caching mode")]
```

Comment:
This is an awful lot of digging into a data structure.  Is that how it has to be?

If so, I'd recommend using the thread-first macro to make this easier to read:
```clojure
(-> (workflow "calls") vals first first (get "Effective call caching mode"))
;; or
(-> (vals (workflow "calls")) first first (get "Effective call caching mode"))
```
Depending on where you think the logical starting place of the chain is.